Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM rust:1.66 as builder
+
+# Set working directory
+WORKDIR /usr/src/rpc
+
+# Copy source code
+COPY . .
+
+# Build the application
+RUN cargo build --all --release
+
+# Create a new container from scratch to reduce image size
+FROM debian:buster-slim
+
+# Install any necessary dependencies
+RUN apt-get update && apt-get install -y libssl-dev ca-certificates && rm -rf /var/lib/apt/lists/*
+
+# RUN update-ca-certificates
+
+# Set the working directory
+WORKDIR /usr/src/app
+
+# Copy the built binary from the previous container
+COPY --from=builder /usr/src/rpc/target/release/kakarot-rpc .
+
+# Expose the port that the RPC server will run on
+EXPOSE 3030
+
+# Run the binary
+CMD ["./kakarot-rpc"]
```

Comment:
though this is fine, there is apparently a best practice consisting in using `tiny`, see for example https://github.com/eqlabs/pathfinder/blob/4ab915a830953ed6f02af907937b46cb447d9a92/Dockerfile#L120

Reply:
ok!